app/views/football_matches/index.html.erb

```diff
@@ -0,0 +1,26 @@
+
+<table>
+  <thead>
+    <tr>
+      <th>Match Date</th>
+      <th>Home Team</th>
+      <th>Score Home</th>
+      <th>Score Away</th>
+      <th>Away Team</th>
+      <th></th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% football_matches.each do |football_match| %>
+      <tr>
+        <td><%= l(football_match.match_date, format: :date_with_time) %></td>
+        <td><%= football_match.home_team.name %></td>
+        <td><%= football_match.home_team_score || 0 %></td>
+        <td><%= football_match.away_team_score || 0 %></td>
+        <td><%= football_match.away_team.name %></td>
+        <td><%= link_to "Place Bet" if football_match.match_date > (DateTime.now + 1.hour) %></td>
```

Comment:
I though we discussed moving `football_match.match_date > (DateTime.now + 1.hour)` out of this file ?